app/pages/project/index.jsx

```diff
@@ -303,15 +311,43 @@ class ProjectPageController extends React.Component {
     });
   }
 
-  handlePreferencesChange(key, value) {
+  handleProjectPreferencesChange(key, value) {
     const changes = {};
     changes[key] = value;
-    const { preferences } = this.state;
-    if (preferences) {
-      preferences.update(changes);
-      this.setState({ preferences });
+    const { projectPreferences } = this.state;
+    if (projectPreferences) {
+      projectPreferences.update(changes);
+      this.setState({ projectPreferences });
       if (this.props.user) {
-        preferences.save();
+        projectPreferences.save();
+      }
+    }
+  }
+
+  handleSplitWorkflowAssignment(projectPreferences, splits) {
+    if (splits && splits['workflow.assignment']) {
+      const projectSetWorkflow = (splits['workflow.assignment'].variant
```

Comment:
Can the `workflow.assignment` key be renamed to `workflowAssignment`? No big deal if it can't, it would just make the key easier to work with in JavaScript.

Reply:
It can't since that is what the key is in the SplitVariant resource itself. Michael Parrish setup the split key naming convention that way and I'm not sure why. It may make more sense in ruby.